### ISSUE TITLE
Uglify is deprecated, but still shown as an example

### ISSUE THREAD

**Issue by rowild** [webpack/webpack#7153]
I get an error when using Uglify:
```Error: webpack.optimize.UglifyJsPlugin has been removed, please use config.optimization.minimize instead.```
The docu still shows it as an example:
```https://webpack.js.org/concepts/plugins/```
Being new to webpack, this confuses me a lot... What to do?

**Comment by PlayMa256** [comment]
Please, open an issue https://github.com/webpack/webpack.js.org here or send a pr to update the docs!
Thanks!

**Comment by rowild** [comment]
Isn't this an issue?

**Comment by PlayMa256** [comment]
No, this is correct, just docs are not updated, the removal of webpack.optimize.uglifyjsplugin was announced during the v4 release.
